README.md

```diff
@@ -100,9 +100,9 @@ The [Salesforce Code Analyzer v5.x (Beta)](https://developer.salesforce.com/docs
 The `forcedotcom/run-code-analyzer@v1` GitHub Action is based on [Salesforce Code Analyzer v4.x](https://developer.salesforce.com/docs/platform/salesforce-code-analyzer/guide/code-analyzer-3x.html), which is the original `@salesforce/sfdx-scanner` Salesforce CLI plugin.
 
 **Note:**
-> We plan to stop supporting v4.x of Code Analyzer in the coming months. We highly recommend that you start using v5.x, which is currently in Beta. For information on v5.x, see https://developer.salesforce.com/docs/platform/salesforce-code-analyzer/guide/code-analyzer.html.
+> We have stopped supporting v4.x of Code Analyzer. We highly recommend that you start using v5.x. For information on v5.x, see https://developer.salesforce.com/docs/platform/salesforce-code-analyzer/guide/code-analyzer.html.
```

Comment:
Is it worth keeping this documentation?

Reply:
I believe we need to keep the pre-5 docs around for a while due to legal reasons.  But I'll mark them Deprecated.

Reply:
Here's a suggested edit of the new note, but please run this by @johnbelosf  who has the final say on this sort of thing.  For example, I guessed on the May 1st date!

As of May 1, 2025, we no longer support v4.x of Code Analyzer.  You should use v5.x of Code Analyzer instead. See https://developer.salesforce.com/docs/platform/salesforce-code-analyzer/guide/code-analyzer.html.  For information about migrating from v4.x to v5.x, see https://developer.salesforce.com/docs/platform/salesforce-code-analyzer/guide/migrate.html.